research/picai/fl_nnunet/nnunet_client.py

```diff
@@ -329,6 +334,18 @@ def setup_client(self, config: Config) -> None:
                 device=self.device,
             )
 
+            # Need to modify num_epochs before initializing so that LRScheduler
+            # recieves the right number of epochs
+            # This will anneal LR to near zero each round
+            # Default nnunet behaviour is do decrease LR every 250 steps
+            # Maybe LRScheduler should just be an argument
```

Comment:
If this is a note to self type question, there is a special comment tag of TODO: Maybe ... that has some nice properties with vscode where you can count how many are in the code etc. if you want to use it